plugins/source/cloudflare/docs/index.md

```diff
@@ -1,54 +0,0 @@
-## Cloudflare Plugin
```

Comment:
I think we should keep the docs directory with this `index.md` file at least (I believe it gets used to populate the website as well). 

We don't need `docs.go` or the `tables` directory anymore though.

Reply:
Done.